opentelekomcloud/resource_opentelekomcloud_smn_topic_v2_test.go

```diff
@@ -42,6 +42,26 @@ func TestAccSMNV2Topic_basic(t *testing.T) {
 	})
 }
 
+func TestAccSMNV2Topic_schemaProjectName(t *testing.T) {
```

Comment:
You're not trying to create SMN topic for another project. Please add second project name for that.